src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -18,25 +18,19 @@ const { educationDetails } = fullSchema.properties;
 
 const { school } = educationDetails;
 
-const {
-  name: schoolName,
-  address: schoolAddress
-} = school.oneOf[0].schoolInformation.properties;
+const { schoolInformation } = school.oneOf[0];
 
-const {
-  street: schoolStreet,
-  street2: schoolStreet2,
-  city: schoolCity,
-  state: schoolState,
-  country: schoolCountry,
-  postalCode: schoolPostalCode
-} = schoolAddress.properties;
+const domesticSchoolAddress = schoolInformation.oneOf[0];
+const internationalSchoolAddress = schoolInformation.oneOf[1];
+
+const countries = domesticSchoolAddress.properties.country.enum.concat(internationalSchoolAddress.properties.country.enum); // TODO access via default definition
+const countryLabels = domesticSchoolAddress.properties.country.enumNames.concat(internationalSchoolAddress.properties.country.enumNames); // TODO access via default definition
 
 const {
   onBehalfOf,
   fullName,
   serviceAffiliation,
-  serviceBranch,
+  // serviceBranch,
```

Comment:
should this be removed entirely? or is it coming back as part of addressing the TODO below?

Reply:
It can be addressed once this PR is merged: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/238